contracts/utils/convex/ConvexStakingWrapper.sol

```diff
@@ -168,27 +168,27 @@ contract ConvexStakingWrapper is ERC20, AccessControl {
         }
 
         //update user integrals for cvx
-        uint256 accountsLength = _accounts.length;
-        for (uint256 u = 0; u < accountsLength; u++) {
-            //do not give rewards to address 0
-            if (_accounts[u] == address(0)) continue;
-            if (_accounts[u] == collateralVault) continue;
-
-            uint256 userI = cvx_reward_integral_for[_accounts[u]];
-            if (_isClaim || userI < cvxRewardIntegral) {
-                uint256 receiveable = cvx_claimable_reward[_accounts[u]] +
-                    ((_balances[u] * (cvxRewardIntegral - userI)) / 1e20);
-                if (_isClaim) {
-                    if (receiveable > 0) {
-                        cvx_claimable_reward[_accounts[u]] = 0;
-                        IERC20(cvx).safeTransfer(_accounts[u], receiveable);
-                        bal = bal - (receiveable);
-                    }
-                } else {
-                    cvx_claimable_reward[_accounts[u]] = receiveable;
+        //do not give rewards to address 0
+        if (_account == address(0) || _account == collateralVault || _account == address(this)) {
```

Comment:
One goal of this change is to make `address(0)` a valid rewards recipient, removing unnecessary code.